Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/InputActionsEditorView.cs

```diff
@@ -221,13 +234,23 @@ public class ViewState
             public int selectedControlSchemeIndex;
             public int selectedDeviceIndex;
         }
+
+        public override void DestroyView()
+        {
+            base.DestroyView();
+
+            Debug.Log("Destroy View");
```

Comment:
Maybe remove method? looks like this is redundant and just for debugging purpose.